analyzer/queries/queries.go

```diff
@@ -453,10 +453,34 @@ var (
     INSERT INTO chain.runtime_transactions (runtime, round, tx_index, tx_hash, tx_eth_hash, fee, gas_limit, gas_used, size, timestamp, method, body, "to", amount, evm_encrypted_format, evm_encrypted_public_key, evm_encrypted_data_nonce, evm_encrypted_data_data, evm_encrypted_result_nonce, evm_encrypted_result_data, success, error_module, error_code, error_message_raw, error_message)
       VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20, $21, $22, $23, $24, $25)`
 
+	RuntimeTransactionUpdate = `
+    UPDATE chain.runtime_transactions
+    SET
+      evm_fn_name = $3,
+      evm_fn_params = $4,
+      error_message = $6,
+      error_params = $7,
+      processed_at = CURRENT_TIMESTAMP
+    WHERE
+      runtime = $1 AND
+      tx_hash = $2`
+
 	RuntimeEventInsert = `
     INSERT INTO chain.runtime_events (runtime, round, tx_index, tx_hash, tx_eth_hash, timestamp, type, body, related_accounts, evm_log_name, evm_log_params, evm_log_signature)
       VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12)`
 
+	RuntimeEventUpdate = `
```

Comment:
```suggestion
	RuntimeEventEvmParsedFieldsUpdate = `
```

or similar, so it's clearer from the name what we're doing. Same for `RuntimeTransactionUpdate`.